index.js

```diff
@@ -42,6 +48,7 @@ module.exports = {
         },
         redisDeployClient: function(context) {
           var redisOptions = this.pluginConfig;
+          redisOptions.port = this.readConfig('port');
```

Comment:
@achambers Every default config value has this issue. Maybe the right path is for ember-cli-deploy to clone the config once per plugin. Anyway, not an issue for this PR in particular.